readme.md

```diff
@@ -83,25 +83,23 @@ Run CLI utility:
 php codecept.phar
 ```
 
-### Composer
-
-```
-php composer.phar require "codeception/codeception"
-```
-
-Read Also [Installation](http://codeception.com/install) | **[QuickStart](http://codeception.com/quickstart)**
+See also [Installation](http://codeception.com/install) | **[QuickStart](http://codeception.com/quickstart)**
 
 ## Getting Started
 
-If you successfully installed Codeception, run this command:
+After you successfully installed Codeception, run this command:
 
 ```
 codecept bootstrap
 ```
 
-this will create a default directory structure and default test suites
+This will create a default directory structure and default test suites.
+
+## Documentation
+
+[Documentation](http://codeception.com/docs/01-Introduction)
 
-See Documentation for more information.
+Documentation is included within the project. Look for it in the ['docs' directory](https://github.com/Codeception/Codeception/tree/master/docs).
```

Comment:
Yes, but master branch stands for the "bleeding edge". Current stable branch is 2.2, once 2.3 is released it will be 2.3, and so on. So there is no permalink for that :(